pages/test_analytics/test_suites.md

```diff
@@ -1,14 +1,13 @@
 # Configuring test suites
 
-In Test Analytics, a test _Suite_ is a collection of tests. A run is to a suite what a build is to a Pipeline.
+In Test Analytics, a test _suite_ is a collection of tests. A run to a suite is what a build to a pipeline is.
```

Comment:
Yeah, I did the initial re-arrangement because I thought the originally sentence was rather clunky. However, since my change isn't much better (i.e. debatable lol), I'll re-write this completely to:

> A suite has a _run_, which is the execution of a test suite. A suite's run is analogous to a pipeline's build.

Since this is the first time in the TA docs that a 'run' is made reference to, I think it's important to call this out as an initial key term in the docs and give it an initial definition.

Reply:
Oh that's much much better 🎉 Brilliant.